azurelinuxagent/common/utils/timeutil.py

```diff
@@ -1,39 +1,20 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the Apache License.
-import datetime
-
-
-def create_timestamp(dt=None):
-    """
-    Returns a string with the given datetime in iso format. If no datetime is given as parameter, it
-    uses datetime.utcnow().
-    """
-    if dt is None:
-        dt = datetime.datetime.utcnow()
-    return dt.isoformat()
-
-
-def create_history_timestamp(dt=None):
-    """
-    Returns a string with the given datetime formatted as a timestamp for the agent's history folder
-    """
-    if dt is None:
-        dt = datetime.datetime.utcnow()
-    return dt.strftime('%Y-%m-%dT%H-%M-%S')
 
+import datetime
 
-def datetime_to_ticks(dt):
```

Comment:
datetime_to_ticks and total_seconds are used only by wire_protocol_data.py, so I moved them there.

Originally I added them here in case they were useful to other code, but that has not been the case